codecov.yml

```diff
@@ -14,5 +14,6 @@ coverage:
     patch: off
 
 ignore:
+  - "Sources/CacheAdvance/FileHandleExtensions.swift"
```

Comment:
Ignoring this file because due to limitations of Github Actions we can't test most of the branches in this file.

Reply:
Can we add a comment to this effect?

Reply:
done!